### ISSUE TITLE
og image path ignores nuxt site config via `runtimeConfig`

### ISSUE THREAD

**Issue by manniL** [nuxt-modules/og-image#139]
https://stackblitz.com/edit/nuxt-ogimage-issues-139?file=nuxt.config.ts

When setting the site config url via `runtimeConfig`, the URL is not reflected correctly.

![image](https://github.com/nuxt-modules/og-image/assets/640208/5f5acf38-ad70-4b23-b405-9655ee445a53)

**Comment by harlan-zw** [comment]
Thanks guessing this is a regression with nuxt-site-config.

While I investigate, I'd use the recommended way to configure it:

```ts
export default defineNuxtConfig({
  site: {
    url: '<site>'
  }
})
```

**Comment by manniL** [comment]
@harlan-zw I guessed so too but didn't have time digging deeper. Used static for now on https://github.com/manniL/lichter.io 👍

**Comment by harlan-zw** [comment]
Have pushed up a fix in 2.2.5 of nuxt-site-config, you may need to clear lock file + node_modules and re-install.
